app/lib/forms/your_employer.rb

```diff
@@ -0,0 +1,30 @@
+module Forms
+  class YourEmployer < Base
+    QUESTION_NAME = :employer_name
+
+    attr_accessor QUESTION_NAME
+
+    validates QUESTION_NAME, presence: true
+
+    def self.permitted_params
+      [QUESTION_NAME]
+    end
+
+    def question
+      OpenStruct.new(
+        type: :text_field,
+        name: QUESTION_NAME,
+        label: I18n.t("registration_wizard.your_employer.label"),
+        hint: I18n.t("registration_wizard.your_employer.hint"),
+      )
+    end
+
+    def next_step
+      :choose_your_npq
+    end
+
+    def previous_step
+      :your_role
+    end
+  end
+end
```

Comment:
No copy at all involved in this question or the views associated with it. It's all stored in the locales file.